docs/source/examples/assignment_workflows/plot_route_choice_set.py

```diff
@@ -0,0 +1,166 @@
+""".. _example_usage_route_choice_generation:
+
+Route Choice set generation
+===========================
+
+In this example, we show how to generate route choice sets for estimation of route choice models, using a
+a city in La Serena Metropolitan Area in Chile.
+
+"""
+
+# %%
+
+# Imports
+from uuid import uuid4
+from tempfile import gettempdir
+from os.path import join
+import numpy as np
+from aequilibrae.utils.create_example import create_example
+# sphinx_gallery_thumbnail_path = 'images/plot_route_choice_set.png'
+
+# %%
+# We create the example project inside our temp folder
+fldr = join(gettempdir(), uuid4().hex)
+
+project = create_example(fldr, "coquimbo")
+
+# %%
+# Choice set generation
+# ---------------------
+
+
+# %%
+
+od_pairs_of_interest = [(71645, 79385), (77011, 74089)]
+nodes_of_interest = (71645, 74089, 77011, 79385)
+
+# %%
+# Let's build all graphs
+project.network.build_graphs()
+# We get warnings that several fields in the project are filled with NaNs.
+# This is true, but we won't use those fields.
+
+
+# %%
+# We grab the graph for cars
+graph = project.network.graphs["c"]
+
+# we also see what graphs are available
+project.network.graphs.keys()
+
+graph.set_graph("distance")
```

Comment:
Are we not setting the utility here?

Reply:
ah we do not care about probabilities because we only want to plot the route set